src/wallet/components/WalletDropdownDisconnect.tsx

```diff
@@ -19,12 +19,12 @@ export function WalletDropdownDisconnect({
       type="button"
       className={cn(
         pressable.default,
-        'relative flex items-center px-4 py-3',
+        'relative flex items-center px-4 pt-3 pb-4',
```

Comment:
Add the extra bottom padding to the disconnect button. This matches the padding for the Identity component.